autoscheduler/frontend/src/components/Schedule/Schedule.css.d.ts

```diff
@@ -0,0 +1,30 @@
+declare namespace ScheduleCssModule {
+  export interface IScheduleCss {
+    "calendar-body": string;
+    "calendar-container": string;
+    "calendar-day": string;
+    "calendar-row": string;
+    calendarBody: string;
+    calendarContainer: string;
+    calendarDay: string;
+    calendarRow: string;
+    header: string;
+    "header-tile": string;
+    headerTile: string;
+    "hour-label": string;
+    "hour-marker": string;
+    hourLabel: string;
+    hourMarker: string;
+    "meetings-container": string;
+    meetingsContainer: string;
+    "v-time-label-margin": string;
+    vTimeLabelMargin: string;
```

Comment:
What's the difference between the camelCase their "kebab-case" counterparts (i.e. `"hour-label"` vs `hourLabel`)?

Reply:
kebab-case is standard for CSS but would be invalid in JS/TS. Therefore, the new loader adds a camelCase version as well, so that our JS/TS files can access the styles. The kebab-case version is left in there by default so that if you wanted to, you could bypass TypeScript type-checking by using `styles[hour-label]`, but `styles.hourLabel` just works better.